Libs/CMakeLists.txt

```diff
@@ -19,8 +19,13 @@ set(SlicerExecutionModel_EXTRA_EXECUTABLE_TARGET_LIBRARIES
   CACHE INTERNAL "SlicerExecutionModel extra executable target libraries" FORCE
   )
 
+if (NOT Slicer_USE_SYSTEM_vtkAddon)
```

Comment:
@jcfr Correct me if I'm wrong, but I'm not sure if we could rely exclusively on vtkAddon_DIR, since it would be already defined by this point.

Slicer_USE_SYSTEM_vtkAddon is still somewhat consistent with the use of Slicer_USE_SYSTEM_VTK/ITK/CTK, etc.

Reply:
VTK/ITK/CTK .. support both `<proj>_USE_SYSTEM` and `<proj>_DIR` options. Setting `<proj>_DIR` means we explicitly provide a location where a config is expected. `<proj>_USE_SYSTEM` is set to ON when `proj` found in any location should be used.

In the case of `vtkAddons`, at that stage we do not support the "use system" approach (e.g there are no `apt-get install vtkAddon` yet)

Reply:
But not worry,  I will revisit the approach while integrating vtkAddon remote module

Reply:
Ok, sounds good. Thanks for the clarification!